ui/raidboss/data/triggers/tsukuyomi-ex.js

```diff
@@ -29,9 +29,9 @@
     },
     {
       id: 'Tsukuyomi Torment',
-      regex: / 14:2BBB:Tsukuyomi starts using Torment Unto Death on (\y{Name})/,
-      regexDe: / 14:2BBB:Tsukuyomi starts using Todesqualen on (\y{Name})/,
-      regexFr: / 14:2BBB:Tsukuyomi starts using Brimade Meurtrière on (\y{Name})/,
+      regex: / 14:(2BBB|2EB2):Tsukuyomi starts using Torment Unto Death on (\y{Name})/,
```

Comment:
Oh, that makes sense, thanks!

You need to make these `(?:2BBB|2EB2)`, so that it doesn't create a matching group.  The alarmText uses matches[1], which it's expecting to be the name match.